app/services/gcp_storage.py

```
@@ -0,0 +1,75 @@
+            gcs_uri = f"gs://{self.bucket_name}/{unique_filename}"
+            logger.info(f"✅ 오디오 파일 업로드 완료: {gcs_uri}")
+            
+            return gcs_uri
+            
+        except Exception as e:
+            logger.error(f"❌ 오디오 파일 업로드 실패: {e}")
+            if 'temp_file_path' in locals():
+                try:
+                    os.unlink(temp_file_path)
+                except:
+                    pass
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(BaseException)`

(SIM105)